middleware/src/main/java/br/ufrn/dimap/middleware/infrastructure/lifecycleManagement/modelo/ILifecycleManagement.java

```diff
@@ -0,0 +1,17 @@
+package br.ufrn.dimap.middleware.infrastructure.lifecycleManagement.modelo;
+
+
+
+/**
+ * The Interface ILifecycleManagement.
+ * 
+ * @author
+ * @version 1.0
+ * @see LifecycleManagement Interface
+ */
+public interface ILifecycleManagement {

```

Comment:
Change interface name to LifecycleManagement, following the pattern used by the others classes and interfaces